### PR TITLE
Small change of reusable workflow api

### PR DESCRIPTION
**Pull request by fbeutin-ledger**

### CODE REVIEW

File: .github/workflows/build_and_functional_tests.yml [hunk 1]
```diff
@@ -20,15 +20,14 @@ on:
 jobs:
   build_application:
     name: Build application using the reusable workflow
-    uses: LedgerHQ/ledger-app-workflows/.github/workflows/reusable_build.yml@v1
+    uses: LedgerHQ/ledger-app-workflows/.github/workflows/reusable_build.yml@fbe/more_options
```
Comment (fbeutin-ledger) [inline .github/workflows/build_and_functional_tests.yml:23]: will be moved to v1 after the `fbe/more_options` branch merge

File: .github/workflows/build_and_functional_tests.yml [hunk 2]
```diff
@@ -20,15 +20,14 @@ on:
...
     with:
-      upload_artifact: true
-      uploaded_artifact_name: compiled_app_binaries
+      app_binaries_artifact: compiled_app_binaries
 
   ragger_tests:
     name: Run ragger tests using the reusable workflow
     needs: build_application
-    uses: LedgerHQ/ledger-app-workflows/.github/workflows/reusable_ragger_tests.yml@v1
+    uses: LedgerHQ/ledger-app-workflows/.github/workflows/reusable_ragger_tests.yml@fbe/more_options
```
Comment (fbeutin-ledger) [inline .github/workflows/build_and_functional_tests.yml:30]: will be moved to v1 after the `fbe/more_options` branch merge

**Review by lpascal-ledger — APPROVED** [review]
Fine by me, although as I said on [this PR](https://github.com/LedgerHQ/ledger-app-workflows/pull/3/files), it is not very clear why `app_binaries-artifact` should be the same.
I think that having the first being `upload_name` and the second `download_name` would make the connection clearer between the two jobs.

**Review by xchapron-ledger — COMMENTED** [review]
can you clean .gitignore in this repo too?

**Comment by fbeutin-ledger** [comment]
> Fine by me, although as I said on [this PR](https://github.com/LedgerHQ/ledger-app-workflows/pull/3/files), it is not very clear why `app_binaries-artifact` value should be the same in the two jobs. The fact that the second depends on the first is implicit. I think that having the first being `upload_name` and the second `download_name` would make the connection clearer between the two jobs.

Done

**Comment by fbeutin-ledger** [comment]
> can you clean .gitignore in this repo too?

Done

**Review by xchapron-ledger — APPROVED** [review]

**Pull request merged**